ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -82,9 +84,11 @@ const getIgvTracks = (igvSampleIndividuals, individualsByGuid, sampleTypes) => {
               type: 'merged',
               name: track.name,
               height: track.height,
+              updated: minJunctionEndsVisible,
               tracks: [coverageTrack, track],
             }
           }
+          track.updated = minJunctionEndsVisible
```

Comment:
I'm confused why you are setting a property called `updated` on the track - from the TGG viewer code it looks like `minJunctionEndsVisible` is an option that should be passed directly to the track itself - https://github.com/broadinstitute/tgg-viewer/blob/master/src/redux/selectors.js#L182

Reply:
The IGV.js doesn't update the display even when the track options are changed. The IGV component in seqr has to update (or refresh) the display after the options of the tracks are changed. Currently, only deleted and added tracks are refreshed by the IGV component. I added a new option `updated`. The IGV component will refresh the track if the value of the `updated` option is changed. The IGV.js doesn't care about any unknown options.
The TGG-Viewer's junction visibility options aren't working at least on my computer. I don't have a chance to check with Ben yet.

Reply:
I think it would be better to instead define a constant list of track properties to check to see if they changed and check those for reloading, rather than sending a meaningless extra property which can only support one property for update, especially since you never actually set the `minJunctionEndsVisible` property on the track itself. Something like
```
const TRACK_UPDATE_PROPERTIES = ['minJunctionEndsVisible']
...
const prevTracksById = prevProps.tracks.reduce((acc, track) => ({ ...acc, [getTrackId(track)]: track }), {})
const prevTrackIds = Object.keys(prevTracksById)
...
tracks.forEach((track) => {
  const prevTrack = track.name && prevTracksById[track.name]
  if (prevTrack) {
    const shouldUpdate = TRACK_UPDATE_PROPERTIES.some(property => track[property] !== prevTrack[property])
    if (shouldUpdate) {
      this.browser.removeTrackByName(track.name)
      this.browser.loadTrack(track)
    }
  }
})
```


> The TGG-Viewer's junction visibility options aren't working at least on my computer.

We can't add a control panel to seqr that does not work! If this is not working then this absolutely can not be deployed, and in fact you should not have PR'd a change until that issues is resolved.
That said, since you never actually add the property to the junction track, that might be why its not working, so I would try updating as I suggested and seeing if that fixes the issue